### ISSUE TITLE
Server download doesn’t work

### ISSUE THREAD

**Issue by Sillymanblah** [Iskallia/Vault-public#839]
**Bug Description**
Not entirely sure what the bug is, but when running the downloadable server files that you can get from Curseforge, just trying to run the server (I am using Java’s latest JDK-17.0.2) it doesn’t even reach the eula before it crashes. 

**To Reproduce**
1. Download the mod pack server download from Curseforge. 
2. Download the latest Java runtime environment. (In this case JDK-17.0.2)
3. Extract server files to a location to run the server and make sure the port used isn’t in use (for me I had to swap: 25569 -> 25566)
4. Run the server (either through a cmd prompt or through directly clicking the forge-1.16.5 executable file. 

**Expected behavior**
About 10 or so lines appear in the cmd prompt, before it crashes and closes out, if you run the file manually you can look at these and they are all forge and/or Java errors with directories. (I can add a photo or put source here if needed)

**Screenshots or Crash Report**
None available at this time

**Info (please complete the following information):**
- World Type: Server
- The_Vault Version: 1.12.4

**Additional context**
Running off of a Windows 10 computer, seems to be one class/module out of place crashing the server. Which is weird because usually Minecraft just routes around it. Problem occurs before EULA because set either true or false this error occurs.

**Comment by Robinlub** [comment]
Hi Sillymanblah,

Delete all the versions of java you have and install java jdk 15. that fixed it for me

**Comment by Sillymanblah** [comment]
Do I need to delete the other versions or can I just force the server to use 15? Because I have other servers running on the JDK 17

**Comment by Sillymanblah** [comment]
I downloaded JDK 15.0.2, and the server ran, thank you so much!
**NOTE**: it is not required that you delete all other Java versions, but if you don’t, you have to create a command that calls upon that specific version of Java. 

A typical Java command looks like:
java -Xmx# -Xms# -jar [forge location]

You can replace the java part with the JDK 15.0.2 location, typically formatted:
“C:\Program Files\Java\jdk-15.0.2\bin\java.exe”
then follow it with the rest:
-Xmx# -Xms# -jar [server location]\forge 1.16.5-36.2.23.jar

**TL;DR**
If you try running a server and have issues, the Java version that you should obtain is JDK 15, and you can leave other Java versions on your computer as long as you call upon the correct one in the commands running the server.
